raiden-ts/tests/unit/epics/transport.spec.ts

```diff
@@ -1828,3 +1836,1716 @@ describe('transport epic', () => {
     });
   });
 });
+
+describe('transport epic', () => {
+  const matrixServer = 'matrix.raiden.test'; // define later in fixture constant
+  const partnerRoomId = `!partnerRoomId:${matrixServer}`;
+  const processed: Processed = {
+    type: MessageType.PROCESSED,
+    message_identifier: makeMessageId(),
+  };
+  const fetch = jest.fn(async () => ({
```

Comment:
Since some time ago, `fetch` is provided as part of the `mocks.ts`, you don't need/shouldn't re-create and re-assign it here. If you don't need the `json` method, you can make it optional in the type declaration there.

Reply:
Also made the changes